### DISCUSSION TITLE
SMTP Basic Auth support ends Sept. 2025 for Microsoft Exchange Online - Must Zitadel react?

### DISCUSSION THREAD

**Discussion by michaelwitt13** [zitadel/zitadel#8843] [Q&A]
As the title says, Microsoft plans to stop support for SMTP with Basic Authentication. Basic Authentication are Base64 encoded username and password credentials. 
As Zitadel's SMTP provider configuration (generic or outlook.com) wants username and password (mandatory fields?), I wonder if Zitadel must implement a new feature to configure MS SMTP providers with alternative authentication methods if Microsoft Exchange Online should be used for sending e-mails!

Here are the sources:
https://techcommunity.microsoft.com/t5/exchange-team-blog/exchange-online-to-retire-basic-auth-for-client-submission-smtp/ba-p/4114750
https://www.borncity.com/blog/2024/07/29/exchange-online-basisauthentifizierung-fr-client-bermittlung-smtp-auth-endet-sept-2025/

**Comment by hifabienne** [comment]
Hi @michaelwitt13 
We have not yet looked into this.
But thanks for raising the awareness.

**Reply by michaelwitt13** [reply]
Hi @hifabienne ,
After more than 6 months: are there any news regarding SMTP Auth for Microsoft servers?
